### ISSUE TITLE
Nativescript FilePicker for ios

### ISSUE THREAD

**Issue by Shifs** [kazemihabib/nativescript-FilePicker#2]
I want to perform file (pdf) upload functionality in my application (Android and ios). I see that filepicker is currently available only for Android . Is there any way by which I can use it for ios?

**Comment by kazemihabib** [comment]
Yes , this dialog is just for android because I used some java code in it.like this one https://github.com/kazemihabib/nativescript-FilePicker/blob/master/app/dialogs/file_picker/file_picker_dialog.ts#L64 
just port them to objective-c or use nativescript Files api.
As I don't use nativescript any more I can't update this.

**Comment by Shifs** [comment]
Thanks . I also wanted to know If I want to create a plugin to connect
Nativescript with NativeMapSDK since I have never made a plugin, I don't
know how much time can it take. So can you please tell me how much time
estimate it can take?




*Regards,*

Shifa Salheen
Lead Programmer Analyst
Argusoft India Limited.
www.argusoft.com

**Comment by kazemihabib** [comment]
nativescript changes a lot in these months that I didn't follow it and I can't tell you how much it might take I think it's better if you ask it here : https://nativescriptcommunity.slack.com
